Sources/Core/Internal/BundleTextureImageLoader.swift

```diff
@@ -8,34 +8,27 @@ import Foundation
 import CoreGraphics
 
 internal final class BundleTextureImageLoader: TextureImageLoader {
-    private let bundle: Bundle
+    private let bundle: BundleProtocol
 
-    init(bundle: Bundle = .main) {
+    init(bundle: BundleProtocol = Bundle.main) {
         self.bundle = bundle
     }
 
-    func loadImageForTexture(named name: String, scale: Int) -> CGImage? {
+    func loadImageForTexture(named name: String, scale: Int, format: TextureFormat) -> CGImage? {
+        guard let extensionName = format.extensionName else {
```

Comment:
This doesn't seem like a valid case, and removing the `unknown` case from the enum would fix that 🙂